src/core_plugins/kibana/public/home/components/add_data.js

```diff
@@ -209,16 +214,35 @@ const AddDataUi = ({ apmUiEnabled, isNewKibanaInstance, intl }) => {
             </EuiLink>
           </EuiText>
         </EuiFlexItem>
+        <EuiFlexItem className={footerItemClasses}>
```

Comment:
This link needs to be added conditionally since `ML plugin` will not always be installed or enabled.

Also, why is the csv import in the ML plugin? Maybe the CSV import needs to be put into its own plugin.

Reply:
@nreese Would this entail doing something like:
```
const mlEnabled = chrome.getInjected('mlEnabled');
...
if (mlEnabled === true) {<EuiFlexItem ... />}
```

My intent here was to get the ball rolling on this change, but I'm getting a little out of my depth :) Thanks for the feedback!

Reply:
I think I'm getting close with something like 
```
{mlEnabled !== false ? <EuiFlexItem>...</EuiFlexItem> : null}
```

Reply:
yes, but you will have to add `mlEnabled ` to the injected vars. [mlEnabled](https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/index.js#L32) is added by the APM application but that may not be enabled/installed either. I think the best bet is to add `mlEnabled` to https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/index.js and then check that value for displaying the FlexItem